src/Microsoft.VisualStudio.Editors/PropPages/AdvBuildSettingsPropPage.resx

```diff
@@ -718,9 +718,15 @@
     <value>Microsoft.VisualStudio.Editors.PropertyPages.PropPageUserControlBase, Microsoft.VisualStudio.AppDesigner, Version=42.42.42.42, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</value>
   </data>
   <data name="CSharpLanguageVerison.Default" xml:space="preserve">
-    <value>C# latest major version (default)</value>
+    <value>C# latest version with preview opt-in (default)</value>
```

Comment:
What is the difference between this and "default"?

Reply:
This is "default"?